TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -35,6 +36,11 @@ object TeleOpMovement : Component {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // if right stick y is moved use the movement of the linear slide
+        if (gamepad.right_stick_y != 0f) {
```

Comment:
What happens here if you accidentally brush the joystick? Or if it gets drift? Perhaps it would be better to check if the `right_stick_y` is outside of a range.

Reply:
I was unsure what to change it to made it 0.2 but if that is not enough let me know.

Reply:
Yeah, we'll need to test if `0.2` is a good number or if it needs to be decreased.